src/main/java/com/jpop/productservice/controller/ProductServiceController.java

```diff
@@ -35,11 +38,23 @@ public ProductServiceController(ProductInsertService productInsertService, Produ
         this.productDeleteService = productDeleteService;
     }
 
+    @ApiOperation(value = "Get all products", notes = "This will be used to get all the products in inventory")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Product found in inventory"),
+            @ApiResponse(code = 404, message = "URL not found"),
+            @ApiResponse(code = 400, message = "Bad request")
+    })
     @GetMapping(value = "/", produces = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<List<Product>> getAllProducts() {
         return ResponseEntity.ok(productDetailService.getAvailableProducts());
     }
 
+    @ApiOperation(value = "Insert new products", notes = "This will be used to add products in inventory")
+    @ApiResponses(value = {
+            @ApiResponse(code = 201, message = "Product added in inventory"),
```

Comment:
It's better to specify the response header `location` in 201 resposne

Reply:
I am not sure if I have done in the correct way now. I am showing URI class in header. Please advise

Reply:
That's right.